storage/Cargo.toml

```diff
@@ -4,7 +4,27 @@ version = "0.1.0"
 edition = "2021"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+
 [dependencies]
-rocksdb = { version = "0.19", package = "ckb-rocksdb" }
-sea-orm = { version = "0.11", features = ["runtime-tokio-native-tls", "sqlx-all"] }
-sparse-merkle-tree = { version = "0.6", features = ["trie"] }
+anyhow = "1.0"
+async-trait = "0.1"
+blake2b-rs = "0.2"
+dashmap = "5.4"
+ethereum-types = { version = "0.14", features = ["arbitrary", "codec", "rlp", "serialize", "std"] }
+lazy_static = "1.4"
+parking_lot = "0.12"
+rocksdb = { package = "ckb-rocksdb", version = "0.19", default-features = false, features = ["snappy", "march-native"] }
+sea-orm = { version = "0.11", features = ["runtime-tokio-rustls", "sqlx-sqlite", "macros"], default-features = false }
```

Comment:
We always use `native_tls` feature, because of the `rustls`  has some problem in cross-platform.